kinder/ci/kubeadm-periodic.tests.md

```diff
@@ -44,35 +44,31 @@ kinder is doing, that is to use an existing CI/Release build.
 
 Upgrade tests are meant to verify the proper functioning of the `kubeadm upgrade` workflow. Following upgrade tests are verified:
 
-| from                                         | e.g.              | to                                     | e.g.             |
-| -------------------------------------------- | ----------------- | -------------------------------------- | ---------------- |
-| current<br />(release/stable)                | v1.14.1           | master<br />(ci/latest)                | v1.15.0-alpha... |
-| current -1/minor<br />(ci/latest-1.13) | V1.13.6-alpha...  | current<br />(release/latest-1.14)     | v1.14.1          |
-| current -2/minor<br />(ci/latest-1.12)       | V1.12.9-alpha...  | current -1/minor<br />(ci/latest-1.13) | V1.13.5          |
-| current -3/minor<br />(ci/latest-1.11)       | V1.11.10-alpha... | current -2/minor<br />(ci/latest-1.12) | V1.12.8          |
-
-TODO: currently tests are not consistent with regards to the selection of from/to versions (some stable-->ci,
-others ci-->latest). Define if/how to rationalize
+| from                                    | e.g.              | to                                     | e.g.             |
+| --------------------------------------- | ----------------- | -------------------------------------- | ---------------- |
+| current -1/minor<br />(ci/latest-1.15)  | v1.15.1-alpha...  | current<br />(ci/latest-1.16)          | v1.16.0-alpha... |
```

Comment:
[1] ci/latest 1.16 --> ci/latest*

with the following note:
* please note that for current we are testing ci/latest and not ci/latest-1.16. This is because the 1.16 branch is aligned only periodically until the release is cut. The back-side of this is that at a certain point master will become 1.17 even if 1.16 is not actually out; to make this work, kubeadm upgrade is required to use the -f flag to ignore the error for skipping one version, but this is an acceptable trade off